src/pages/en/guides/integrations-guide/prefetch.md

```diff
@@ -110,7 +106,6 @@ export default {
 
 ## Troubleshooting
 
-*   If your installation doesn't seem to be working, make sure to restart the dev server.
```

Comment:
See previous note re: troubleshooting - amend the wording to not say BE SURE TO, but still leave as troubleshooting guidance?